dev/archery/archery/utils/codec.py

```diff
@@ -48,6 +48,8 @@ def encode(b):
             "unit": b.unit,
             "less_is_better": b.less_is_better,
             "values": b.values,
+            "time_unit": b.time_unit,
```

Comment:
Odd. Why is this file in `dev/archery/archery/utils/` when all other archery benchmark code is in `dev/archery/archery/benchmark/`.

Reply:
I have no idea :-) Feel free to move it around if it's only used by `archery.benchmark`.

Reply:
Cool, I'll move it then.